spring-boot-project/spring-boot-dependencies/pom.xml

```diff
@@ -1003,6 +1004,11 @@
 				<artifactId>validation-api</artifactId>
 				<version>${javax-validation.version}</version>
 			</dependency>
+			<dependency>
```

Comment:
I don't know what version of `master` you have but the current master [already has this dependency management](https://github.com/spring-projects/spring-boot/blob/c27fa7bf91cd00aa12fad7dd6d60c2b64ac12247/spring-boot-project/spring-boot-dependencies/pom.xml#L1044)